pkg/apis/core/types.go

```diff
@@ -3508,6 +3512,14 @@ type LoadBalancerIngress struct {
 	// (typically AWS load-balancers)
 	// +optional
 	Hostname string
+
+	//The list has one entry per service port in the manifest
```

Comment:
nit: comments want a " " between `//` and text

Reply:
Godoc style starts with the field name

Reply:
We need to tighten the spec.  This field is optional, but if it is provided, there must be one entry per service port, mapped by port number and protocol (avoid the word "manifest, tooooo vague).

Reply:
> Godoc style starts with the field name

[Here](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#edit-typesgo) I read this

> Comments for types should not contain the type name; API documentation is generated from these comments and end-users should not be exposed to golang type names.